packages/vaadin-material-styles/version.js

```diff
@@ -4,18 +4,24 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 
+import { defineCustomElement } from '@vaadin/component-base/src/define.js';
```

Comment:
As we now import this helper, we also need to add `@vaadin/component-base` dependency to `package.json`. Same applies to Lumo.

Reply:
Yep, I pushed an update to fix this too.